engines/bastion_katello/app/assets/javascripts/bastion_katello/errata/views/apply-errata-confirm.html

```diff
@@ -119,7 +119,7 @@ <h3 ng-show="errata" translate>Apply {{ errata.errata_id }}</h3>
     <div class="checkbox" ng-show="updates.length > 0">
       <label>
         <input name="noDepSolve" ng-model="noDepSolve" type="checkbox"/>
-        <span translate>Skip dependency solving for a significant speed increase. If the update cannot be applied to the host, delete the incremental content view version and retry the application with dependency solving turned on.</span>
+        <span translate><b>Skip dependency solving for a significant speed increase</b>. If the update cannot be applied to the host, delete the incremental content view version and retry the application with dependency solving turned on.</span>
```

Comment:
Not a 100% sure if translation would work here..Maybe something like this?
<b><span translate>....</span></b><span ...../>

Reply:
I see two options to fix this. Either I split up the string into a <b translate> block and a <span translate> block, or I just bold the entire thing.
Bolding the entire thing would change the look of course. Splitting it into two blocks would make the strings re-require retranslation.
Unless there's a potential fix that I'm not seeing where we can selectively bold and keep the translation string together?

Reply:
![image](https://github.com/user-attachments/assets/ebe98e5c-99da-43d7-b509-0811b23d2385)

![image](https://github.com/user-attachments/assets/6c8ae5fa-c344-410c-8871-02fe554c677f)